### ISSUE TITLE
typo and possible error

### ISSUE THREAD

**Issue by jaygray0919** [ampproject/amp.dev#5114]
for https://amp.dev/documentation/examples/visual-effects/animated_snackbar/?format=websites#asd

/s/dissappear/disappear

also, the inline example (for the target page) does not work as advertised in the sample code (it does not disappear according to the terms in the code).
in contrast, playground works as advertised

**Comment by patrickkettner** [comment]
fixed typo in d17751b

snackbar _does_ work, but you need to scroll way down and way back up - might be some kind of optimization that is only triggering the animation if something is in the viewport?

**Comment by jaygray0919** [comment]
agree - it does not work as advertised in the sample code; it has different behavior.

**Comment by patrickkettner** [comment]
absolutely! your report was correct, just wanted to add some context for anyone looking at a fix
